File: TM 3 Functions_Modules_Packages/Functions_Modules_Packages/program3.py
# Tek Raj Joshi
# Superset ID: 1368453

def get_factorial(num):
    if num == 0:
        return 1
    fact = 1
    while num > 0:
        fact = fact * num
        num -= 1
    return fact
    

integer = int(input("Enter an integer: "))
print(get_factorial(integer))

File: TM 15 C#-Basics/C#-Basics/Data Types/program3.cs
// Tek Raj Joshi
// Superset ID: 1368453

using System;
public class Program3
{
   public static void Main( )
    {
        Console.Write("Enter the temperature in Celsius: "); 
        int celsius = Convert.ToInt32(Console.ReadLine());
 
        Console.WriteLine("Kelvin = {0}", celsius + 273);
        Console.WriteLine("Fahrenheit = {0}", celsius * 18 / 10 + 32);
    }
}